quick-start.md

```diff
@@ -2,222 +2,87 @@
 
-1. Install Kontena CLI tool.
-2. Create a [Kontena Platform](using-kontena/platform.md).
-3. Install [Kontena Nodes](using-kontena/nodes.md).
-4. Deploy first [Kontena Stack](using-kontena/stacks.md).
+1. Sign-up to Kontena Cloud.
+2. Create an organization for Kontena Platforms.
+3. Create a [Kontena Platform](using-kontena/platform.md).
+4. Install [Kontena Nodes](using-kontena/nodes.md).
+5. Deploy first [Kontena Stack](using-kontena/stacks.md).
 
 Follow these steps to get started with Kontena on Kontena Cloud quickly. If you like to tinker and maintain the Kontena Platform yourself, please follow the [slow start](./slow-start.md) guide.
 
-## Step 1. Install Kontena CLI (command-line interface)
+## Step 1. Sign-up to Kontena Cloud
 
-### MacOS (OSX)
+You can sign-up to Kontena Cloud by following this link: https://cloud.kontena.io/sign-up
 
-You can install Kontena CLI using our [official installer](https://gh-releases.kontena.io/kontena/kontena/pkg/latest) or [Homebrew](https://brew.sh/) :
+## Step 2. Create an Organization
 
-```
-$ brew install kontena
-```
-
-### Debian / Ubuntu
-
-You can install Kontena CLI using our [official deb package](https://gh-releases.kontena.io/kontena/kontena/deb/latest).
-
-### Linux / Windows
-
-> Prerequisites: You'll need Ruby version 2.1 or later installed on your system. For more details, see the official [Ruby installation docs](https://www.ruby-lang.org/en/documentation/installation/).
-
-You can install the Kontena CLI using the Rubygems package manager (which is included in Ruby).
-
-```
-$ gem install kontena-cli
-```
+After you have signed up and verified your email, you can create an organization for your project. An organization is a workspace for development teams to monitor, manage and operate Kontena Platforms.
 
-After the installation is complete, you can test the installation by checking the Kontena CLI version with `kontena version`.
+<img src="_images/kontena-cloud-create-org.png" alt="Create Organization" />
 
-## Step 2. Create a Kontena Platform
+## Step 3. Create a Kontena Platform
 
-The easiest (and preferred) way to provision Kontena Platform Master is to use the built-in Kontena Cloud Platform provision feature of Kontena CLI. In this guide, we will provision Kontena Platform to Kontena Cloud and nodes to the local development environment using [Vagrant](https://www.vagrantup.com/). If you want to install nodes to some other environment, please see [Installing Kontena](using-kontena/install-nodes/) documentation.
+Kontena Platform will abstract all available compute resources and make these resources available to your container workloads. To create your first Kontena Platform, just open the 'Create Platform' dialog and fill in the platform name. For testing purposes `Mini` platform type is enough.
```

Comment:
The Kontena Platform will abstract all available compute resources and make these resources available to your container workloads. To create your first Kontena Platform, just open the 'Create Platform' dialog and fill in the platform name. For testing purposes `Mini` platform type is enough.